### PR TITLE
Add BaseEnum::getKeys

### PR DESCRIPTION
**Pull request by Soullivaneuh**
The goal is to be able to have the constants key list directly.

Will add a not on upgrade once #7 will be merged.

### CODE REVIEW

File: src/BaseEnum.php [hunk 1]
```diff
@@ -42,6 +42,24 @@ public static function getConstants()
     }
 
     /**
+     * Returns constants keys.
+     *
+     * @param callable|null $callback A callable function compatible with array_map
+     *
+     * @return string[]
+     */
+    final public static function getKeys($callback = null)
```
Comment (greg0ire) [inline src/BaseEnum.php]: Can you add an example in the README please ?

Comment (Soullivaneuh) [inline src/BaseEnum.php]: Done.

**Comment by Soullivaneuh** [comment]
Not sure if I have to add a note on `UPGRADE-2.0.md` because there is no BC break here.

**Comment by greg0ire** [comment]
UPGRADE is for bc breaks only

**Comment by greg0ire** [comment]
Wonderful! Thanks!

**Pull request merged**